ci/Jenkinsfile-scan

```
@@ -1,52 +1,32 @@
+
+
 def veracodeCredentials = usernamePassword(
         credentialsId: 'veracode-api-credentials',
         passwordVariable: 'VERACODE_KEY',
         usernameVariable: 'VERACODE_ID')
 
-def slackChannel = 'components-ci'
+// Job variables declaration
 
-final String tsbiImage = 'jdk11-svc-springboot-builder'
-final String tsbiVersion = '3.0.8-2.6-20220928070500'
-
-// Pod definition
-final String podDefinition = """\
-apiVersion: v1
-kind: Pod
-spec:
-    containers:
-        -
-            name: main
-            image: 'artifactory.datapwn.com/tlnd-docker-dev/talend/common/tsbi/${tsbiImage}:${tsbiVersion}'
-            command: [cat]
-            tty: true
-            volumeMounts: [
-                {name: docker, mountPath: /var/run/docker.sock},
-                {name: efs-jenkins-connectors-se-m2, mountPath: /root/.m2/repository}
-            ]
-            resources: {requests: {memory: 3G, cpu: '2'}, limits: {memory: 8G, cpu: '2'}}
-    volumes:
-        -
-            name: docker
-            hostPath: {path: /var/run/docker.sock}
-        -
-            name: efs-jenkins-connectors-se-m2
-            persistentVolumeClaim: 
-                claimName: efs-jenkins-connectors-se-m2
-    imagePullSecrets:
-        - name: talend-registry
-""".stripIndent()
+// Maven package skipped elements
+// Everything that is not needed for the veracode scan is excluded to reduce time
+_SKIP_OPTS = "--define maven.test.skip=true\
+              --define spotless.apply.skip=true\
+              --define checkstyle.skip=true\
+              --define rat.skip=true\
+              --define skipTests=true\
+              --define invoker.skip=true"
```

This will generate a string with a lot of white space which can clutter the logs.
I like to build these strings like this:

```suggestion
_SKIP_OPTS = [
  '--define maven.test.skip=true',
  '--define spotless.apply.skip=true',
  '--define checkstyle.skip=true',
  '--define rat.skip=true',
  '--define skipTests=true',
  '--define invoker.skip=true'
].join(' ')
```

YMMV

---

I like your solution very much!